src/gpu/intel/jit/gemm/generator/pieces/copy_plan.cpp

```diff
@@ -2589,6 +2589,12 @@ void CopyPlan::optimizeConcatenate(bool initial)
             if (!initial)
                 doJoin &= (i1.simd + i2.simd <= 32);
 
+            if (hw >= HW::XeHPC)
+                // Packed SIMD2 byte mov seems to have a bug.
```

Comment:
nit: Maybe make this comment slightly more specific: "... seems to have a bug when dst offset is odd"